src/modules/settings/flow-typed/UserSettings.js

```diff
@@ -0,0 +1,46 @@
+/* @flow */
+
+export type UserSettings = {
```

Comment:
Why all options have any type by default?

Reply:
@bautrukevich it's input settings, we actually do not know what type we will receive. There is also type `Settings` with all types defined. But I think we can use just `$Shape<Settings>` instead of `UserSettings`. I'll investigate it.

Reply:
Thank you for your explain 👍

Reply:
@bautrukevich oh, there is difference between `Settings` and `UserSettings`. First one is internal settings object, i.e. `tabs` is array, not string. And the second one is the input settings object, where `tabs` should be string, but actually user can pass anything else.